### PR TITLE
feat: Api methods to list test and test suites with executions

### PR DESCRIPTION
**Pull request by vsukhin**
## Pull request description 

Api methods to list test and test suites with executions

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by Toxer10 — APPROVED** [review]

File: api/v1/testkube.yaml [hunk 1]
```diff
@@ -117,6 +117,35 @@ paths:
...
+        - $ref: "#/components/parameters/TextSearch"
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/TestSuiteWithExecution"
```
Comment (nicufk) [inline api/v1/testkube.yaml:139]: is TestSuiteWithExecution defined in the api schema?

Comment (vsukhin) [reply api/v1/testkube.yaml:139]: yes, it was done on previous pr

Comment (nicufk) [reply api/v1/testkube.yaml:139]: Now i see it was defined before

**Review by nicufk — APPROVED** [review]
LGTM, Good Job!

**Pull request merged**